src/microbe_stage/Microbe.cs

```diff
@@ -2566,7 +2566,7 @@ private void CheckBinding()
 
         // We check here if we can bind, not to waste computational power with Invoke,
         // although we'll still have to recheck it later.
-        if (IsBindingForbidden(other))
+        if (CanBindToMicrobe(other))
```

Comment:
You are!

Reply:
Yeah, I renamed it but forgot to account for the change in the logic. Don't code at 2am, folks. 
(I'm waiting for the checks to be done and I'll request review then)